WatchDogCore/WatchDog/src/nl/tudelft/watchdog/core/logic/event/eventtypes/EventType.java

```diff
@@ -17,6 +17,46 @@
 	
```

Comment:
It's not clear from the name of this class, nor from its location in the package structure that it deals with DebugEvents only.